lib/network/generate_key.go

```diff
@@ -0,0 +1,156 @@
+package sebaknetwork
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+var (
+	host     = "localhost"
+	validFor = time.Duration(1000000)
+	rsaBits  = 2048
```

Comment:
2048 starts to be on the weak side, the default those days is 4096
Also, we should set a less arbitraty duration, and document how many days / months / parsec it correspond to.